.eslintrc

```diff
@@ -1,26 +1,32 @@
 {
-  "parser": "babel-eslint",
-  "extends": [
-    "standard",
-    "standard-react"
-  ],
-  "env": {
-    "es6": true
-  },
-  "plugins": [
-    "react"
-  ],
-  "parserOptions": {
-    "sourceType": "module"
-  },
-  "rules": {
-    // don't force es6 functions to include space before paren
-    "space-before-function-paren": 0,
-
-    // allow specifying true explicitly for boolean props
-    "react/jsx-boolean-value": 0,
-
-    // allow comma-dangle in multiline objects & arrays
-    "comma-dangle": ["error", "only-multiline"]
-  }
-}
+   "parser": "@typescript-eslint/parser",
+   "extends": [
+     "standard",
+     "standard-react",
+     "plugin:@typescript-eslint/recommended"
```

Comment:
Extending this rule is probably forcing 4 spaces for indent. Add this line in rules section to make tab size 2 spaces:

    "indent": ["error", 2],

See https://stackoverflow.com/a/48906878/9321755 for more details.